config/testgrids/openshift/redhat-openshift-ocp-release-4.13-informing.yaml

```diff
@@ -540,6 +540,33 @@ dashboards:
     results_url_template:
       url: https://prow.ci.openshift.org/job-history/<gcs_prefix>
     test_group_name: periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-upgrade-aws-ovn-arm64
+  - base_options: width=10&exclude-filter-by-regex=Monitor%5Cscluster&exclude-filter-by-regex=%5Eoperator.Run%20template.*container%20test%24
```

Comment:
Can you move the config/testgrids/openshift/... changes to another PR? Those appear to be unrelated to the VPA presubmit tests

Reply:
Hi @laoj2 ,
I did not add or make any changes to config/testgrids/openshift/. When I raised the PR initially I could only see my changes in a single file. Now I see 3 files got changed as part of this PR. 

No idea why config/testgrids/openshift/ changes are pulled in into this PR..